### PR TITLE
Features 20200205.v1

### PR DESCRIPTION
**Pull request by eoliveros**
ZAPD-21: add searching by invoiceid/txid/recipient/sender to the 'Transactions' table

### CODE REVIEW

File: zapd/models.py [hunk 1]
```diff
@@ -511,6 +511,7 @@ def decline_view(self):
         # process the proposal
         if proposal.status == proposal.STATE_CREATED:
             proposal.status = proposal.STATE_DECLINED
+            proposal.authorizer = current_user
```
Comment (djpnewton) [inline zapd/models.py]: this seems unrelated?

File: zapd/admin.py [hunk 2]
```diff
@@ -19,8 +19,8 @@
 admin.add_view(UserModelView(User, db.session))
 admin.add_view(ProposalModelView(Proposal, db.session))
 admin.add_view(RestrictedModelView(Category, db.session))
-admin.add_view(RestrictedModelView(Transaction, db.session))
 admin.add_view(RestrictedModelView(CreatedTransaction, db.session))
+admin.add_view(TransactionRestrictedModelView(Transaction, db.session))
```
Comment (djpnewton) [inline zapd/admin.py:22]: dont change the order of the tabs

File: zapd/models.py (hunk 1 above)
Comment (eoliveros) [reply zapd/models.py]: Was originally trying to include the username in the authorized by when declined as well. Old stuff.... deleting it.

File: zapd/admin.py (hunk 2 above)
Comment (eoliveros) [reply zapd/admin.py:22]: changed the order.
